core/src/main/java/com/linecorp/armeria/client/DefaultClientFactory.java

```diff
@@ -171,4 +175,19 @@ public void close() {
     void doClose() {
         clientFactoriesToClose.forEach(ClientFactory::close);
     }
+
+    private URI normalizeUri(URI uri, Scheme scheme) {
+        if (isUndefinedUri(uri)) {
+            // See https://github.com/line/armeria/pull/2241 discussion for more details.
+            return uri;
+        }
+        if (scheme.serializationFormat() != SerializationFormat.NONE) {
```

Comment:
Would add a line break since there is a logical break in what we're doing.

Reply:
Done